packages/lodestar/src/eth1/impl/ethers.ts

```diff
@@ -95,6 +96,14 @@ export class EthersEth1Notifier implements IEth1Notifier {
     this.logger.verbose("Eth1 notifier stopped");
   }
 
+  /**
+   * Sync calls this after it's up to head.
+   * @param blockHash block hash of last eth1 vote
+   */
+  public async collectEth1Data(blockHash: string): Promise<void> {
+    this.checkpoint = (await this.getBlock(blockHash)).number;
+  }
```

Comment:
@wemeetagain How about we introduce event hub (eventFeed?!) in your document. I think it would be really helpful if we had some central place that all modules can depend on to receive some lodestar events suchs as sync statuses, processed blocks etc?